File: app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/customcontrols/gamepad/GamepadDataProvider.java
package net.kdt.pojavlaunch.customcontrols.gamepad;


import net.kdt.pojavlaunch.platform.input.PlatformGrabListener;

public interface GamepadDataProvider {
    GamepadMap getMenuMap();
    GamepadMap getGameMap();
    boolean isGrabbing();
    void attachGrabListener(PlatformGrabListener grabListener);
}

File: app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/customcontrols/gamepad/GamepadDataProvider.kt
package net.kdt.pojavlaunch.customcontrols.gamepad

import git.artdeell.dnbootstrap.glfw.GrabListener

interface GamepadDataProvider {
    fun getMenuMap(): GamepadMap
    fun getGameMap(): GamepadMap
    fun isGrabbing(): Boolean
    fun attachGrabListener(grabListener: GrabListener)
}
